pandas/tests/computation/test_eval.py

```diff
@@ -115,6 +118,18 @@ def lhs(request):
 midhs = lhs
 
 
+@pytest.fixture
+def idx_func_dict():
+    return {
```

Comment:
maybe worth a note that this is used in combination with the c_idx_type fixture; there might also be a way to make this build on top of that (fine for a future PR if possible)

Reply:
Ah nice find. Yeah appears we can eliminate a fixture by combining both